docs/Feeds.md

```diff
@@ -443,6 +484,28 @@ Collector: Generic URL Fetcher (`intelmq.bots.collectors.http.collector_http`)
 
 Parser: Missing
 
+
+## Netlab 360
+
+### DGA Feed
+
+Status: Unknown
+
+Collector: Generic URL Fetcher (`intelmq.bots.collectors.http.collector_http`)
+ * http_url: http://data.netlab.360.com/feeds/dga/dga.txt
+
+Parser: Netlab 360 DGA (`intelmq.bots.parsers.netlab_360.parser_dga`)
+
+### Magnitude EK Feed
+
+Status: Unknown
```

Comment:
AFAIU the upstream description (http://data.netlab.360.com/ek), this feed lists URLs with exploits? Please be more detailed here.